src/shared/views/Mapper/Tile.js

```diff
@@ -191,9 +233,9 @@ export class _Tile extends React.Component<Props> {
                     key={`touch-${taskId}`}
                     source={imageSource}
                 >
-                    <View style={[styles.tileOverlay, { backgroundColor: overlayColor }]} key={`view-${taskId}`}>
-                        {animatedRows}
-                    </View>
+
+                    {comp}
```

Comment:
I guess "comp" refers to completeness

Reply:
It's "component", declared on line 207, and defined on 210 or 218. We could probably rename it and add typing annotations...